.travis.yml

```diff
@@ -50,15 +55,27 @@ before_install:
 - makepanda/selfdestruct.py --yes
 
 install:
-- virtualenv --python=${PYTHON_INTERP:-python3.4} venv && source venv/bin/activate
+- >
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then 
+    ./.travis/install_macos.sh;
+  else
+    virtualenv --python=${PYTHON_INTERP:-python3} venv;
+  fi
```

Comment:
I would rather either put both platforms in separate scripts or put both platforms in the .travis.yml.  I'm strongly leaning towards having everything in .travis.yml.

Reply:
The reason I was hesitant on that was because it might be a pain to edit shell scripts embedded inside a .yml file, but you're probably right that having a bunch of small files littered about would be even more of one.